js/state-from-store-mixin.jsx

```diff
@@ -140,16 +150,11 @@ var StateFromStore = function(stateDescriptors) {
                 changeListeners: {}
             };
 
-            var initialState = {};
-            Object.keys(stateDescriptors).forEach(function(stateKey) {
-                var stateData = fetchNewStateData(
-                    this, stateKey, false).stateData;
-                initialState[stateKey] = stateData;
-            }, this);
-            return initialState;
+            return getStoreState(this);
         },
 
         componentDidMount: function() {
+            this.setState(getStoreState(this));
```

Comment:
Nooo don't do this, it causes a re-render if the state changes! Maybe we could move this to `componentWillMount`, so we don't cause this to happen?